targets/tofino/fuzzer.cpp

```diff
@@ -134,14 +153,27 @@ InitialConfig TofinoTnaFuzzer::produceInitialConfig() {
             continue;
         }
         auto table = tables.Get(tableId);
+        if (table.match_fields_size() == 0 || table.is_const_table()) {
+            continue;
+        }
+        /// TODO: Remove this when Flay supports RANGE.
```

Comment:
Do we not support range in the bfruntime parser? I can add it.

Reply:
We don't. In "core/control_plane/bfruntime/protobuf.cpp:produceTableMatch", only exact, lpm and ternary are supported. Same for P4runtime.

Reply:
Will fix this.